modules/QnA/tests/QnA.html

```diff
@@ -22,12 +22,18 @@
 	kWidget.featureConfig({
 		'targetId': 'kaltura_player',
 		'wid': '_1802381',
-		'uiconf_id' : '30139111',
+		'uiconf_id' : '25906281',
```

Comment:
@OrenMe If you will review the file changes you will see the I reverted all my changes in the state configuration so Einat will not be affected.

Most of the changes were made on the mw.cuePoints but I think that I managed to make specific modifications. 

Lets review it together tomorrow.